i18n/it/cosmic_settings.ftl

```diff
@@ -27,212 +27,216 @@ import = Importa
 light = Chiaro
 mode-and-colors = Modalità e colori
 recent-colors = Colori recenti
-reset-default = Reimposta predefinito
-reset-to-default = Reimposta predefinito
+reset-default = Ripristina predefinito
+reset-to-default = Ripristina predefinito
 rgb = RGB
-window-hint-accent = Colore di suggerimento della finestra attiva
-window-hint-accent-toggle = Usa il colore d'accento del tema come suggerimento della finestra attiva
+window-hint-accent = Colore d'accento per la finestra attiva
+window-hint-accent-toggle = Usa il colore del tema come colore d'accento
 
-auto-switch = Passa automaticamente dalla modalità Chiara a quella Scura
-    .desc = All'alba passa automaticamente alla modalità Chiara
+auto-switch = Cambia automaticamente del tema schiaro al tema scuro
+    .desc = All'alba, cambia al tema Chiaro
```

Comment:
DAMN!
Thanks for spotting it, it's just one typo, right?

"schiaro"... in my defense, I was seeing double, literally.

Reply:
"cambia al tema chiaro" is correct since is says that after the sun rises it changes back to light theme.
It makes sense.

Is the second typo the capital C?

Reply:
I think "del" should be "dal"

Reply:
You're thinking correctly... I'm still wondering how tired I was this afternoon to have made such mistakes.